From 792756b37de8a9dd5da60602c6db44fd1e591fbd Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 13 Apr 2023 21:12:01 +0100
Subject: [PATCH 1/3] gnu: packages music: Delete use-module duplicates.

* gnu/packages/music.scm: Delete duplicated (guix build-system qt).
---
 gnu/packages/music.scm | 2 --
 1 file changed, 2 deletions(-)

diff --git a/gnu/packages/music.scm b/gnu/packages/music.scm
index 69fe7009be..10281a6a12 100644
--- a/gnu/packages/music.scm
+++ b/gnu/packages/music.scm
@@ -84,11 +84,9 @@ (define-module (gnu packages music)
   #:use-module (guix build-system qt)
   #:use-module (guix build-system scons)
   #:use-module (guix build-system glib-or-gtk)
-  #:use-module (guix build-system qt)
   #:use-module (guix build-system waf)
   #:use-module (guix build-system trivial)
   #:use-module (guix build-system go)
-  #:use-module (guix build-system qt)
   #:use-module (gnu packages)
   #:use-module (gnu packages admin)
   #:use-module (gnu packages algebra)
-- 
2.39.2

